data/src/main/kotlin/com/alfresco/content/data/AppConfigUtils.kt

```diff
@@ -143,7 +148,38 @@ val gson: Gson = GsonBuilder()
     .enableComplexMapKeySerialization()
     .create()
 
+@RequiresApi(Build.VERSION_CODES.O)
 private val formatter = DateTimeFormatterBuilder()
     .append(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
     .optionalStart().appendOffset("+HHMM", "Z").optionalEnd()
     .toFormatter()
+
+// Function to store a JSON object
+fun saveJsonToSharedPrefs(context: Context, key: String, obj: Any) {
+    val sharedPreferences: SharedPreferences = PreferenceManager.getDefaultSharedPreferences(context)
+    val editor = sharedPreferences.edit()
+    val gson = Gson()
+
+    // Convert object to JSON
+    val jsonString = gson.toJson(obj)
```

Comment:
Add try catch in case of invalid json

Reply:
obj is non-null and it will have values always

Reply:
This object remains valid at all times since it is a model class parsed from the API response.